### PR TITLE
CNI portmap plugin fixes and improvements

### PR DESCRIPTION
**Pull request by aojea**
The portmap plugin can use external chains for SNAT

> externalSetMarkChain - string, default nil. If you already have a Masquerade mark chain (e.g. Kubernetes), specify it here. This will use that instead of creating a separate chain. When this is set, markMasqBit must be unspecified.

https://github.com/containernetworking/plugins/tree/v0.8.3/plugins/meta/portmap

v0.8.4 CNI portmap fixes  a bug that was causing flakiness in the k8s CI

It also bumps the base image version with a newer version of iptables user space binaries

### CODE REVIEW

File: pkg/build/node/cni.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 The default CNI manifest and images are our own tiny kindnet
 */
 
-var defaultCNIImages = []string{"kindest/kindnetd:0.5.3@sha256:bc1833b3da442bb639008dd5a62861a0419d3f64b58fce6fb38b749105232555"}
+var defaultCNIImages = []string{"aojea/kindnetd:0.5.4"}
```
Comment (neolit123) [inline pkg/build/node/cni.go]: should a new image be pushed to `kindest/kindnet`?

Comment (BenTheElder) [reply pkg/build/node/cni.go]: yeah, discussed this PR in the kind meeting today, holding it pending #1124 fix and then pushing a small kind release with assorted small fixes after .6

then this, and also upgrading the debian-ipatables base to the new one coming in k8s 1.17 with much newer iptables

**Comment by aojea** [comment]
/hold
https://github.com/kubernetes-sigs/kind/pull/1132#discussion_r352911665

**Comment by aojea** [comment]
/hold cancel
the small release was already done

**Comment by aojea** [comment]
/retest
forget to push the new image for testing

**Comment by aojea** [comment]
/hold
we need to wait for CNI plugins 0.8.4 release

**Comment by BenTheElder** [comment]
- can we downgrade to a version without the bug while we wait for v0.8.4 to be released?

**Comment by aojea** [comment]
> * can we downgrade to a version without the bug while we wait for v0.8.4 to be released?

🤷‍♂ this bug was there since the beginning

**Comment by BenTheElder** [comment]
sorry will need an update, I did some heavy work on deps today which overlapped (but yay updated deps and smaller images!)

please send me a reminder when the CNI plugins is out and we'll try to get this in ASAP to get the flake fix :-)

**Comment by aojea** [comment]
we only need the new portmap plugin, but we have to wait until the CNI folks cut a new release with the fix I did to avoid the concurrency issues that we see in the CI

**Pull request closed without merging**